backends-common/rabbitmq/src/test/java/org/apache/james/backends/rabbitmq/RabbitMQTest.java

```diff
@@ -118,13 +125,19 @@ private Boolean containerIsRestarted(DockerRabbitMQ rabbitMQ) {
 
         private String createQueue(Channel channel) throws IOException {
             channel.exchangeDeclare(EXCHANGE_NAME, DIRECT_EXCHANGE, DURABLE);
-            String queueName = channel.queueDeclare().getQueue();
+            String queueName = channel.queueDeclare(UUID.randomUUID().toString(), DURABLE, !EXCLUSIVE, AUTO_DELETE, ImmutableMap.of()).getQueue();
```

Comment:
Can you extract `UUID.randomUUID().toString()` & `ImmutableMap.of()` to explain what these parameters are?